app/sass/login-form.scss

```diff
@@ -0,0 +1,72 @@
+$base-color: #b73535;
+@mixin login-form-font($size) {
+  font: {
+    family: Georgia, Cambria, "Times New Roman", Times, serif;
+    style: italic;
+    size: $size;
+  };
+}
+.login-form {
+  margin-top: 100px;
+  display: flex;
+  justify-content: center;
+  input {
+    background: none;
+    border: solid 2px $base-color;
+    color: $base-color;
+    padding: 15px 40px;
```

Comment:
This hasn't been mentioned before, but I'll add it to readme - as Liberator is intended for mobile use as well, we'll be better with `rem`s instead of `px`. Basics can be read here: http://snook.ca/archives/html_and_css/font-size-with-rem